File: parte001/ex014_numero_mayor_menor_igual.cpp
// Hacer un algoritmo que, dados dos valores numéricos A y B, escriba un mensaje diciendo 
// si A es mayor, menor o igual a B.

#include <iostream>

using namespace std;

int main()
{
    int a, b;
    cout << "Ingrese el valor de A: ";
    cin >> a;
    cout << "Ingrese el valor de B: ";
    cin >> b;

    cin.ignore();

    if (a > b)
    {
        cout << "A es mayor que B" << endl;
    }
    else if (a < b)
    {
        cout << "A es menor que B" << endl;
    }
    else
    {
        cout << "A es igual a B" << endl;
    }

    return 0;
}

File: parte001/ex014_numero_mayor_menor_igual.cs
// Hacer un algoritmo que, dados dos valores numéricos A y B, escriba un mensaje diciendo 
// si A es mayor, menor o igual a B.

using System;

namespace Parte001
{
    class Program
    {
        static void Main(string[] args)
        {
            int a, b;

            Console.Write("Ingrese el valor de A: ");
            a = Convert.ToInt32(Console.ReadLine());

            Console.Write("Ingrese el valor de B: ");
            b = Convert.ToInt32(Console.ReadLine());

            if (a > b)
            {
                Console.WriteLine("A es mayor que B");
            }
            else if (a < b)
            {
                Console.WriteLine("A es menor que B");
            }
            else
            {
                Console.WriteLine("A es igual a B");
            }

            Console.ReadKey();
        }
    }
}
